dipy/reconst/dki.py

```diff
@@ -380,8 +380,10 @@ def _F2m(a, b, c):
 
 
 def directional_diffusion(dt, V, min_diffusivity=0):
-    r""" Calculate the apparent diffusion coefficient (adc) in each direction of
-    a sphere for a single voxel [1]_
+    r"""Compute apparent diffusion coefficient (adc).
+
+    Calculate the apparent diffusion coefficient (adc) in each direction of
+    a sphere for a single voxel :footcite:t:`NETOHENRIQUES201585`.
```

Comment:
This is a bit of an eyesore. Maybe we could we use a uniform format for bibtex keys? For example, "NetoHenriques2015NeuroImage"

Reply:
> This is a bit of an eyesore

ahahah, you are right

> Maybe we could we use a uniform format for bibtex keys?

I agree, even if the bibtex file is big. I like the beginning of your example but I do not think we should finish with the journal. so just {firstname}{name}{year}{letter | optional if multiple paper} is that ok ?

Reply:
Yeah. That would work well.